### PR TITLE
Refactoring owner entity

### LINKED ISSUE #92: Refactoring owner entity
## This task is a part of the issue:
 - #64 

## Description
After a team discussion, it was decided to modify the owner's structure and add a city property. It is necessary to refactor all the code utilizing this structure based on this change.

### Dev requirements and restrictions:
```swift
struct Owner {
    /// Uniquely identifies the pet owner.
    let id: UUID

    /// The first name of the pet owner.
    var name: String

    /// The last name of the pet owner.
    var surname: String

    /// The phone number of the pet owner.
    var phone: String

    /// URL that contains an avatar image of the pet owner.
    var avatarURL: URL?

    /// The owners city.
    var city: String

    /// The address of the pet owner, street and buidling.
    var address: String

    /// The rating of the pet owner, that should be provided in range 0 to 5.
    var rating: Double
}
```

### PR DESCRIPTION
**Pull request by iBYarchitect**
## Summary
This PR is a part of the work to resolve #92 

## Changes
Changes have affected the view used by the owner entity. The owner model itself has also been modified, including the dummy owner, and tests have been edited.

## Checks to complete
- [x] The PR title has a conventional commit style (short and descriptive).
- [x] The PR description includes essential information that helps understand the purpose of the changes.
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] My code follows our style guide.
- [x] I've performed a self-review of my code.
- [x] Every non-private interface is documented properly.
- [x] I've added tests for my code (if it makes sense).
- [x] I've updated `CHANGELOG.md` file.
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: Woof/Woof/Model/Owner/Owner.swift [hunk 1]
```diff
@@ -19,6 +19,9 @@ struct Owner: Identifiable, Codable, Equatable {
     /// URL that contains an avatar image of the pet owner.
     var avatarURL: URL?
 
+    /// The owners city.
```
Comment (NadzeyaShpakouskaya) [inline Woof/Woof/Model/Owner/Owner.swift:22]: nit
```suggestion
    /// The owner's city.
```

File: Woof/Woof/View/OwnerFlow/OwnerProfileView/EditOwnerInformationView.swift [hunk 2]
```diff
@@ -59,13 +65,15 @@ struct EditOwnerInformationView_Previews: PreviewProvider {
                 name: .constant("Emily"),
                 surname: .constant("Thompson"),
                 phone: .constant("527637624567"),
+                city: .constant("My twon"),
```
Comment (NadzeyaShpakouskaya) [inline Woof/Woof/View/OwnerFlow/OwnerProfileView/EditOwnerInformationView.swift:68]: nit
```suggestion
                city: .constant("My town"),
```

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM, some minor typos

File: Woof/Woof/View/OwnerFlow/OwnerProfileView/EditOwnerInformationView.swift (hunk 2 above)
Comment (iBYarchitect) [reply Woof/Woof/View/OwnerFlow/OwnerProfileView/EditOwnerInformationView.swift]: fixed with d4ea5cc

**Pull request merged**